style.css.map

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
I don't think the map file needs to be committed for version control :) But it's OK to have it.